lib/app_profiler/request_parameters.rb

```diff
@@ -16,6 +16,11 @@ def autoredirect
       query_param("autoredirect") || profile_header_param("autoredirect")
     end
 
+    def async
+      val = query_param("async")
+      val == "true" || val == "1"
```

Comment:
We typically don't do strict value checking like this. Is there a reason we want only 1 or true and not _any_ truthy value?